web/crux/src/interceptors/helper.interceptor.ts

```diff
@@ -14,17 +14,19 @@ type GrpcCallLog = {
 @Injectable()
 export default class InterceptorGrpcHelperProvider {
   mapToGrpcObject(context: ExecutionContext): GrpcCallLog {
-    const request = context.getArgByIndex<AccessRequest>(0)
+    const request = context.getArgByIndex(0)
     const surfaceCall = context.getArgByIndex<ServerSurfaceCall>(2)
     return this.mapServerCallToGrpcLog(request, surfaceCall)
   }
 
-  mapServerCallToGrpcLog(request: AccessRequest, surfaceCall: ServerSurfaceCall): GrpcCallLog {
+  mapServerCallToGrpcLog(request: any, surfaceCall: ServerSurfaceCall): GrpcCallLog {
```

Comment:
Can we avoid to using the `any`?

Reply:
As the AccessRequest was removed, we can not assume any other type than `any`